internal/endpoint/machineService.go

```diff
@@ -43,15 +52,23 @@ func (h *endpointHandler) NewMachineService() *restful.WebService {
 
 	ws.Route(ws.POST("/report/{id}").
 		To(h.Report).
-		Doc("report machine by ID").
+		Doc("report machine").
 		Param(ws.PathParameter("id", "identifier of the machine").DataType("string")).
 		Metadata(restfulspec.KeyOpenAPITags, tags).
 		Reads(domain.Report{}).
-		Writes(domain.BootResponse{}).
-		Returns(http.StatusOK, "OK", models.V1MachineResponse{}).
+		Returns(http.StatusOK, "OK", nil).
 		Returns(http.StatusNotAcceptable, "Not acceptable", nil).
 		Returns(http.StatusInternalServerError, "Error", nil))
 
+	ws.Route(ws.POST("/reboot/{id}").
```

Comment:
Who is the potential caller of this endpoint ?

Reply:
If the primary disk check fails for any reason metal-hammer calls this endpoint by also changing boot order from PXE back to HD